plugin/src/main/scala/chisel3/internal/plugin/BundleComponent.scala

```diff
@@ -189,6 +194,49 @@ private[plugin] class BundleComponent(val global: Global, arguments: ChiselPlugi
       elementsImpl
     }
 
+    def generateAutoTypename(bundle: ClassDef, thiz: global.This): Option[Tree] = {
+      // 1: Get immediate defined args of the Bundle
+      // 2: Determine an appropriate naming strategy based on the collection of elements:
+      //    - 1 numeric property only: Generate a name like MyBundle2
+      //    - Multiple numeric properties: Generate a name with word-like descriptors like MyBundle_${width}width
+      //    - Mixed elements: Numeric properties first, type names of Aggregate-likes after, like MyBundle_${width}property_${childBundle.typeName}
```

Comment:
```suggestion
      //    - Multiple numeric properties: Generate a name with word-like descriptors like MyBundle_${width}width
      //    - Mixed elements: Numeric properties first, type names of Aggregate-likes after, like MyBundle_${width}property_${childBundle.typeName}
```

Is it really putting the word `width` and `property` in there? Isn't the example just showing it with the number there?